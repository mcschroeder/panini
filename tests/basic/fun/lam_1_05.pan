
f33 : int -> {w:int|false}
f33 = \x:int. 1

f43 : {v:int|true} -> {w:int|false}
f43 = \x:int. 1

f53 : {v:int|v=1} -> {w:int|false}
f53 = \x:int. 1

f63 : {v:int|v=2} -> {w:int|false}
f63 = \x:int. 1
