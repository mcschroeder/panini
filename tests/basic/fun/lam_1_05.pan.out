tests/basic/fun/lam_1_05.pan:3:1-4: error: unable to verify type of f33
tests/basic/fun/lam_1_05.pan:6:1-4: error: unable to verify type of f43
tests/basic/fun/lam_1_05.pan:9:1-4: error: unable to verify type of f53
tests/basic/fun/lam_1_05.pan:12:1-4: error: unable to verify type of f63